src/PortingAssistantExtensionServer/Services/PortingService.cs

```diff
@@ -59,22 +59,20 @@ public ProjectFilePortingResponse PortingProjects(ProjectFilePortingRequest requ
         {
             try
             {
+                if (ProjectPathToDetails.Count == 0)
```

Comment:
check for nulls as well ProjectPathToDetails == null || ProjectPathToDetails.Count == 0